netlify.toml

```diff
@@ -5,6 +5,3 @@
   status = 200
 
 [context.production.environment]
```

Comment:
we could remove this as well, since it's not being used

Reply:
Thanks @mirandaio. Yup, the `[context.production.environment]` is not doing anything.
Do you mean to remove the `status = 200` as well?

Reply:
@LucaFumis just the `[context.production.environment]`